### ISSUE TITLE
SteamOS 2.60 Broke all my games

### ISSUE THREAD

**Issue by ekianjo** [ValveSoftware/SteamOS#473]
First time after the update, none of my games worked, with The Talos Prophecy at least throwing a visible error "cannot create GL context". After a restart I saw a brief "running unattended upgrades" flash before SteamOS shut down. 

Yesterday I was on 2.60 and everything worked fine apparently. 

Today, I restart my system again, and none of my games work anymore. (any game I try to launch shows "launching" then nothing happens and the start button shows "Play" again). 
I don't know why this is happening, because the nVidia driver was not updated apparently, but something went very wrong after the update. 

Not sure if I am the only person with this issue.

**Comment by ekianjo** [comment]
Additional remark: After 2 reboots I could use again my games just like before... but it seems that there's no way to tell when this issue comes back - maybe at next reboot I will get it again. Really wierd.

**Comment by ekianjo** [comment]
Yeah, I get this issue basically at every 3rd reboot for some reason. I am wondering if something is not up with the video driver since 2.60 was released.

**Comment by wbsdty331** [comment]
Try to reinstall?

**Comment by sharkwouter** [comment]
Try pressing ESC while booting and selecting restore SteamOS. It will automatically recover from the backup. This will remove any modifications you have done to SteamOS, but you will keep all your games and savegames.

**Comment by ekianjo** [comment]
No, reinstall is not an option. My install is not broken.

I have isolated the issue (I think). I realized that some games would still run while some others would refuse and complain that "Steam is not running" - so if I logged out and logged in again, the problem actually solved itself. Very, very strange, since i was effectively connected to Steam on LAN.

**Comment by wbsdty331** [comment]
what is your computer configure?
